packages/roosterjs-editor-types/lib/corePluginState/LifecyclePluginState.ts

```diff
@@ -31,7 +32,14 @@ export default interface LifecyclePluginState {
     /**
      * External content transform function to help do color transform for existing content
      */
-    onExternalContentTransform: ((htmlIn: HTMLElement) => void) | null;
+    onExternalContentTransform:
+        | ((
+              element: HTMLElement,
+              fromDarkMode: boolean,
```

Comment:
Should we make the new parameters optional to avoid breaking change? Or it is okay?

Reply:
Normally, adding new parameter should be optional. But here we don't expect this function to be called outside editor core API, but it is a callback that people can pass in. In that case even the passed-in callback doesn't have those new parameters it can still pass the build. So from this part it is not a breaking change.